### ISSUE TITLE
Foundation Fixed Top-Bar

### ISSUE THREAD

**Issue by rahiyansafz** [foundation/foundation-sites#12193]
# Please help me to make it.

### I was trying to make a fixed top bar in my layout with foundation xy-grid. i followed the doc and the building blocks but its not working. I am using Angular 11 but the foundation js is written in jquery.

![Image of Angular]
(https://user-images.githubusercontent.com/70625091/103916505-b879b580-5136-11eb-8694-fb2886939ff7.png)
)
I was trying to make this sidebar and top bar to be fixed. any solutions?

**Comment by DanielRuf** [comment]
It may be better to ask in the community forums at https://foundation.discourse.group/

**Comment by mattbag** [comment]
Yes css position -fixed or absolute- will work out better. You don't need grid elements for layout. You can also use css grid if your support excludes ie11

**Comment by rahiyansafz** [comment]
> Yes css position -fixed or absolute- will work out better. You don't need grid elements for layout. You can also use css grid if your support excludes ie11

it's not working.

**Comment by DanielRuf** [comment]
> it's not working.

Instead of just writing "it's not working" please try to provide a code example as codepen or similar so we can see the actual problem. Otherwise we have to guess what is missing or wrong.

Not sure if you saw it but when you create a new issue there is an issue template that should be filled out with all relevant details: https://github.com/foundation/foundation-sites/issues/new?assignees=&labels=&template=1-bug-report.md
